list.c

```diff
@@ -0,0 +1,57 @@
+#include <stdlib.h>
+#include <stdint.h>
+#include "list.h"
+
+// simple linked list implementation
+
+struct list *list_new(void) {
+    return calloc(1, sizeof(struct list));
+}
+
+void list_clear(struct list *list) {
```

Comment:
we have a common coding style, in which the open bracket is put on a new line, alone. can you please fix this?